com.vcvrack.Rack2.yaml

```diff
@@ -0,0 +1,38 @@
+id: com.vcvrack.Rack2
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: rack
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+
+modules:
+  - name: rack2free
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 metainfo.xml ${FLATPAK_DEST}/share/${FLATPAK_ID}.metainfo.xml
+      - install -Dm755 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644  res/icon.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/com.vcvrack.Rack2.png
+      - install -Dm755 rack_wrapper.sh ${FLATPAK_DEST}/bin/rack
+      - cp -a ./ ${FLATPAK_DEST}/
+    sources:
+      - type: file
+        path: com.vcvrack.Rack2.desktop
+      - type: file
+        path: metainfo.xml
+      - type: script
+        dest-filename: rack_wrapper.sh
+        commands:
+          - cd /app
+          - exec /app/Rack --system /app --user ${XDG_DATA_HOME} $@
+      - type: archive
+        url: https://vcvrack.com/downloads/RackFree-2.5.2-lin-x64.zip
+        sha256: 6c7bba5f3ccf8fe667771ddd2619c6a17f84b5dfd31d3683f56c253e27629ca1
+
+cleanup:
+  - rack_wrapper.sh
```

Comment:
this doesn't do what you think it does.

```suggestion
```

Reply:
It does :) It deletes `rack_wrapper.sh` once it's installed in `/app/bin`  exactly as described in [doc](https://docs.flatpak.org/en/latest/manifests.html), cleanup section.
I'd maybe move it into the module scope but keep it.

Reply:
that because to should never reach there then. cleanup is to remove things installed by the build system (there is none) that are not wished for version reasons.

Reply:
I'm confused: if I'm removing `cleanup` section and run `flatpak  run --command=sh  com.vcvrack.Rack2` I see the `rack_wrapper.sh` in the /app dir. If I keep `cleanup` section - file been removed.

Reply:
It's coming in `/app` because `cp` with `*` is copying it. If you move it before copying, it won't be needed.